src/test/java/seedu/address/model/medicalhistory/DiagnosisTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.model.medicalhistory;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class DiagnosisTest {
+    private final String validDoctorName = "Dr. Zhang";
+    private final String invalidDoctorName = "dr.lame";
```

Comment:
I have an invalidDoctorName in one of my tests. Does it affect unity that we use the same invalid name?